drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -117,6 +117,10 @@ static void copy_pvt(struct gps_pvt *dest, nrf_gnss_pvt_data_frame_t *src)
 		dest->sv[i].elevation = src->sv[i].elevation;
 		dest->sv[i].azimuth = src->sv[i].azimuth;
 		dest->sv[i].signal = src->sv[i].signal;
+		dest->sv[i].in_fix =
+			src->sv[i].flags & BIT(NRF_GNSS_SV_FLAG_USED_IN_FIX);
```

Comment:
BIT() macro can't be used, because the flag is defined by the value, not the number of the bit.

Also the value from the AND operation needs to be compared to the flag value. This would work with a bool, but doesn't seem to work with bit fields.

Reply:
Thanks, I interpreted the docs all wrong here.. Should be fixed now, hopefully.